docs/tutorial/chapter-3-adding-mutations-to-your-api.md

```diff
@@ -160,14 +160,14 @@ schema.extendType({
 ```diff
 Mutation {
 -  signup: User
-+  signup(name:String!, email:String!): User
++  signup(name: String!, email: String!): User
 }
 ```
 
 </div>
 
 1. Add an `args` property to the field definition to define its args. Keys are arg names and values are type specifications.
-2. Use the Nexus helpers for defining an arg type. There is one such helper for every GraphQL scalar such as `schema.intArg` and `schema.booleanArg`. Should you want to reference a type like some InputObject then use `schema.arg({ type: "..." })`.
```

Comment:
Sometimes my writing style is technically correct but might be too hostile toward non-native speakers. Just confirming that its your intention to try and bring down the level to a more approachable one?

Reply:
> Just confirming that its your intention to try and bring down the level to a more approachable one

That's correct. I would also understand if it was irritating for you to see your language being degraded with childish words 🙈. My english is pretty poor in terms of vocabulary and proper literature expressions. Therefore it's likely that some of the expressions are far from being advanced and they don't deserved to be simplified.

- In the case of `anew`, I had to google to make sure it wasn't a typo
- In the case of `Should you want to [...] then`, I frankly don't even understand how that can even make sense 😅

Reply:
Totally makes sense, a simple straight forward international english is absolutely the way to go. Thanks!